File: 6KYU/Highest Scoring Word/solution.cs
using System.Linq;
public static class Kata
{
  private static int ScoreWord(string word)
    => word.Aggregate(0, (acc, i) => acc + (int)i - 96);
  
  public static string High(string s)
    => s.ToLower()
        .Split(" ")
        .Select(w => (Score: ScoreWord(w), Word: w))
        .MaxBy(ws => ws.Score)
        .Word;
}

File: 6KYU/Highest Scoring Word/solution.go
package kata

import "strings"

func ScoreWord(s string) int {
	score := 0
	for _, c := range s {
		score = score + (int(c) - 96)
	}
	return score
}

func High(s string) string {
	words := strings.Split(s, " ")
	res := ""
	highestScore := 0
	for _, word := range words {
		score := ScoreWord(word)
		if score > highestScore {
			highestScore = score
			res = word
		}
	}
	return res
}
